### DISCUSSION TITLE
Git sync failed with colon caracter

### DISCUSSION THREAD

**Discussion by SharkProgramming** [requarks/wiki#3694] [Error / Bug Report]
**Describe the bug**
If you a have an article with the title and/or description with the caracter colon { : } , it wil not be pulled from git because there is a problem with the node .

**To Reproduce**
1. Create an article with title like that : 
Example : example of title
   And/Or Description like that : 
Example : example of description
2. Save the article with content 
3. Activate the git storage and wait for synchronization

**Host Info:**
 - OS: Ubuntu 18.04.3
 - Wiki.js version: 2.0.0-rc.17
 - Database engine: MariaDB 10.3.18

**Comment by KaKi87** [comment]
Duplicate of #3600
